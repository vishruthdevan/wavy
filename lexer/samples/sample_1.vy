aud1 = load("input1.wav")
aud2 = load("input2.wav")

aud1 = fadeIn(aud1, 3.0)
aud2 = fadeOut(aud2, 3.0)

aud1 = trim(aud1, 5.0, 30.0)
aud = trim(aud2, 5.0, 30.0)

res = join(aud1, aud2)

123res = res