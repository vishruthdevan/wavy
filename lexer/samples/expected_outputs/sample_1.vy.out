<IDENTIFIER, "aud1">
<ASSIGN, "=">
<LOAD, "load">
<LPR, "(">
<STRING, "input1.wav">
<RPR, ")">
<IDENTIFIER, "aud2">
<ASSIGN, "=">
<LOAD, "load">
<LPR, "(">
<STRING, "input2.wav">
<RPR, ")">
<IDENTIFIER, "aud1">
<ASSIGN, "=">
<IDENTIFIER, "fadeIn">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<IDENTIFIER, "duration">
<COLON, ":">
<FLOAT, "3.0">
<RPR, ")">
<IDENTIFIER, "aud2">
<ASSIGN, "=">
<IDENTIFIER, "fadeOut">
<LPR, "(">
<IDENTIFIER, "aud2">
<COMMA, ",">
<IDENTIFIER, "duration">
<COLON, ":">
<FLOAT, "3.0">
<RPR, ")">
<IDENTIFIER, "aud1">
<ASSIGN, "=">
<IDENTIFIER, "trim">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<IDENTIFIER, "start">
<COLON, ":">
<FLOAT, "5.0">
<COMMA, ",">
<IDENTIFIER, "end">
<COLON, ":">
<FLOAT, "30.0">
<RPR, ")">
<IDENTIFIER, "aud">
<ASSIGN, "=">
<IDENTIFIER, "trim">
<LPR, "(">
<IDENTIFIER, "aud2">
<COMMA, ",">
<IDENTIFIER, "start">
<COLON, ":">
<FLOAT, "5.0">
<COMMA, ",">
<IDENTIFIER, "end">
<COLON, ":">
<FLOAT, "30.0">
<RPR, ")">
<IDENTIFIER, "res">
<ASSIGN, "=">
<IDENTIFIER, "join">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<IDENTIFIER, "aud2">
<RPR, ")">
<INTEGER, "123">
<IDENTIFIER, "res">
<ASSIGN, "=">
<IDENTIFIER, "res">
<EOF, "">
